core/sql/cli/Cli.cpp

```diff
@@ -10495,13 +10528,18 @@ Lng32 SQLCLI_LOB_GC_Interface
 
       // Restore original data file.
       Int32 rc2=ExpLOBoper::restoreLobDataFile(lobGlobals,tgtLobName, (void *)currContext.exHeap(),hdfsServer,hdfsPort,lobLocation);
+      if (rc2)
+        {
+          lobDebugInfo("restoreLobDataFile Failed",0,__LINE__,lobTrace);
```

Comment:
Is the value of rc2 interesting here? (Or is it always -1 on an error and therefore uninteresting?)